doc/doc-site/docs/welcome.md

```diff
@@ -34,21 +34,26 @@ If you haven't already, consider checking out Econia Labs' [Teach yourself Move
 ## What is Econia?
 
 Econia is a protocol that lets anyone in the world trade a digital asset with anyone else in the world, at whatever price they want.
-More specifically, Econia is an order book, a fundamental financial tool utilized by financial institutions like stock markets, except unlike the New York Stock Exchange or the NASDAQ, Econia is decentralized, open-source, and permissionless.
+More specifically, Econia is an order book, a fundamental financial tool utilized by financial institutions like stock markets, except unlike the New York Stock Exchange or the NASDAQ, Econia is open-source, permissionless, and fully on-chain.
 
 ## Econia v4 is audited
 
-Econia has completed [three independent audits].
+Econia has completed multiple [independent audits].
 
 ## Account addresses
 
-The [Econia repo] uses branches to track the Move package source code published across Aptos Mainnet, Testnet, and Devnet:
+The Econia Move package is persisted indefinitely on both Aptos mainnet and testnet at the following multisig addresses:
```

Comment:
Wouldn't it be nice to be able to also say "immutably?"

"Indefinitely and immutably" sounds very commanding.

(just a thought)